java-sdk/radar-schemas-tools/src/main/java/org/radarcns/schema/registration/KafkaTopics.java

```diff
@@ -85,6 +74,15 @@ public boolean waitForBrokers(int brokers) throws InterruptedException, ZkExcept
                         activeBrokers, brokers, sleep);
             }
         }
+
+        String bootstrapServers = JavaConverters$.MODULE$
+                .seqAsJavaList(zkClient.getAllBrokersInCluster()).stream()
```

Comment:
Is this for conversion between scala and java ? Maybe a comment here can be useful